boards/DIM/Src/App/states/App_DriveState.c

```diff
@@ -175,6 +176,42 @@ static void DriveStateRunOnTick100Hz(struct StateMachine *const state_machine)
         CANMSGS_DIM_SWITCHES_START_SWITCH_ON_CHOICE,
         CANMSGS_DIM_SWITCHES_START_SWITCH_OFF_CHOICE);
 
+    struct RgbLed *board_status_leds[NUM_BOARDS] = {
+        [BMS] = App_DimWorld_GetBmsStatusLed(world),
+        [DCM] = App_DimWorld_GetDcmStatusLed(world),
+        [DIM] = App_DimWorld_GetDimStatusLed(world),
+        [FSM] = App_DimWorld_GetFsmStatusLed(world),
+        [PDM] = App_DimWorld_GetPdmStatusLed(world),
```

Comment:
This is interesting syntax that I've never seen before. Does `[BMS]` just effectively transform `BMS` into a number and insert into the array at that index?

Reply:
Yup that's it! TIL it doesn't seem supported in C++ so that's why it may seem odd to you!? 🤔  

https://eli.thegreenplace.net/2011/02/15/array-initialization-with-enum-indices-in-c-but-not-c